docs/playground.md

```diff
@@ -0,0 +1,252 @@
+* Quick
+* Decent
+* Fair
+* Okay
+* Slow
+* Sluggish
+
+These benchmarks take into account the **visual complexity** of the rule.
+
+
+## Adding a rule to the Rule Board
+
+**Adding a rule to the [Rule Board](../semgrep-app/rule-board/)** allows you to quickly set a rule as part of every Semgrep scan. To add a rule to the Rule Board:
+
+1. Ensure that you are logged-in.
+2. Click **Add to Rule Board**.
+3. Select either the Audit or Block boards depending on the importance of the rule. Matches for rules on the Block board will prevent PR merges within a CI environment.
+
+
+## Saving and sharing a Semgrep rule
+
+A rule is saved by either **explicitly naming it** or when it is **assigned a shortcode** for sharing. Rules are **unlisted **by default. This means they do not appear publicly in the Registry[] nor can they be searched for, but are accessible via their sharing URL.
+
+To explicitly name a rule:
+
+1. Sign into Semgrep.
+2. Click the **Untitled rule** box.
+3. Enter the name of your rule and press **Save**.
+
+To assign a random shortcode:
+
+1. Click **Share**. This will auto-generate a random shortcode and URL to access the rule.
+
+To retrieve or open a rule:
+
+1. Click **File > Open**
+2. Enter the rule's name in the **Filter** search box or scroll to find the rule.
+3. Click on the rule. The rule loads into the Playground.
+
+To rename a rule:
+
+1. Sign into Semgrep. 
+2. In the Playground, open the rule to rename.
+3. Enter a new name and press **Save**. This creates a new rule.
+4. Delete the old rule to complete the renaming process.
+
+
+## Using a rule in CI
+
+Using a rule in your CI pipeline enables you to **run the rule at a certain hook or event**. This allows users to add guardrails against security, style, and general code violations.
+
+Semgrep can be used in the following CI environments:
+
+* GitHub (pre-commit and workflows)
+* GitLab
+* CircleCI
+* AppVeyor
+* TravisCI
+
+**Prerequisites:**
+
+1. Ensure that you have sufficient permissions to update your CI environment.
+
+To add your rule to your CI pipeline:
+
+1. Click **Share**.
+2. Select the CI environment from the tabs under the header **Use in CI**.
+3. Click on the **clipboard icon** on the upper right of the snippet to copy it.
+4. Paste the snippet into the **config file** specified above the snippet.
+![Screenshot of Playground > CI menu](img/playground-use-ci.png "Playground CI menu")
+
+
+## Embedding a rule in your site
+
+The **Embedded Playground** is an interactive editor which can be embedded in an external domain or web page such as a blog post. It is a means to display Semgrep's rules in action.
+
+The Embedded Playground has two panes:
+
+* **Rule Pane.** This displays the rule data. The rule must have a **reference **(either a short identifier or saved name) from Semgrep Playground.
+* **Test Code.** This displays the code that the rule will act upon. On the bottom right is the Run button, which will run the rule on the test code.

```

Comment:
Use the description list here again, please. :-)

Reply:
Done. Have added it to latest commit.